packages/app/src/app/overmind/namespaces/dashboard/state.ts

```diff
@@ -68,13 +69,18 @@ export type State = {
   contributions: Branch[] | null;
   /**
    * v2 repositories (formerly projects)
-   * stores as a record of team id and repositories (or null).
+   * stores as a record of team id and repositories (or undefined).
    * implemented this way to overcome an issue where the
    * delayed synced repositories fetch on a previous team
    * overrides the current team data.
    * @see {@link https://linear.app/codesandbox/issue/XTD-375}
    */
-  repositories: Record<string, Repository[] | null> | null;
+  repositoriesByTeamId: Record<string, Repository[] | undefined>;
+
+  /**
+   * Repository with branches cached based on `team/owner/repo`
+   */
+  repositoriesWithBranches: Record<string, RepositoryWithBranches | null>;
```

Comment:
Why change the type from `undefined` to `null`?

Reply:
I guess it's because of the filter when deleting a repo but that could be changed to a `null` if it returns `undefined`.

Reply:
Looks like a mistake, changed  to `undefined`